instrumentation/elasticsearch/elasticsearch-rest-5.0/javaagent/src/test/groovy/Elasticsearch5RestClientTest.groovy

```diff
@@ -98,18 +98,6 @@ class Elasticsearch5RestClientTest extends AgentTestRunner {
             "${SemanticAttributes.DB_OPERATION.key()}" "GET _cluster/health"
           }
         }
-        span(1) {
-          name expectedOperationName("GET")
-          kind CLIENT
-          childOf span(0)
-          attributes {
-            "${SemanticAttributes.NET_TRANSPORT.key()}" "IP.TCP"
-            "${SemanticAttributes.HTTP_URL.key()}" "_cluster/health"
-            "${SemanticAttributes.HTTP_METHOD.key()}" "GET"
-            "${SemanticAttributes.HTTP_STATUS_CODE.key()}" 200
-            "${SemanticAttributes.HTTP_FLAVOR.key()}" "1.1"
```

Comment:
I think `elasticsearch-rest` is the only instrumentation affected, and it only lost http semantic attributes, which I think is ok because it should be following database semantic conventions (same as `elasticsearch-transport` instrumentation).

I removed the remaining 2 http semantic attributes just now to make it more clear, let me know if this seems better.

Reply:
I agree with Mateusz: this seems like undesired collateral damage

Reply:
does #1822 sound good?

I'm not too fond of mixing db and http semantic conventions on the same span (e.g. people writing span processors may then have to think through this odd case), but I do like the idea of capturing nested http requests as events on the db span.